### PR TITLE
feat: simple Stronghold-based Shimmer claiming flow 

### PR DESCRIPTION
**Pull request by maxwellmattryan**
## Summary

> Please summarize your changes, describing __what__ they are and __why__ they were made.

This PR includes changes to add the happy Stronghold-based Shimmer claiming flow to the onboarding process.

## Changelog

```
- Implement Stronghold-based Shimmer claiming flow
- Add platform API method to copy a file 
- Add util method for creating an account with already bound methods on it
- Add various utils files for sorting, filtering, iterables, etc.
- Add new notification locale entry
- Fix Stronghold recovery password input error reset logic
```

## Relevant Issues

> Please list any related issues using [development keywords](https://docs.github.com/en/issues/tracking-your-work-with-issues/linking-a-pull-request-to-an-issue#linking-a-pull-request-to-an-issue-using-a-keyword).

#3147 

### Platforms

> Please select any platforms where your changes have been tested.

- __Desktop__
  - [ ] MacOS
  - [x] Linux
  - [ ] Windows
- __Mobile__
  - [ ] iOS
  - [ ] Android

### Instructions

> Please describe the specific instructions, configurations, and/or test cases necessary to __test and verify__ that your changes work as intended.

- Try simple case of funds on account 0 address 0
- Try both IOTA- and Shimmer-based Strongholds (only IOTA ones will work)
- Try both old and new Strongholds

## Checklist

> Please tick all of the following boxes that are relevant to your changes.

- [x] I have followed the contribution guidelines for this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [ ] I have added or modified tests that prove my changes work as intended
- [x] I have verified that new and existing unit tests pass locally with my changes
- [x] I have verified that my latest changes pass CI workflows for testing and linting
- [x] I have made corresponding changes to the documentation

### CODE REVIEW

**Review by Tuditi — COMMENTED** [review]
Code looks nice, just a minor suggestion. Will test asap

File: packages/desktop/electron/electronApi.js [hunk 1]
```diff
@@ -323,6 +323,9 @@ const ElectronApi = {
     openUrl: (url) => {
         ipcRenderer.invoke('open-url', url)
     },
+    copyFile: (sourceFilePath, destinationFilePath) => {
+        ipcRenderer.invoke('copy-file', sourceFilePath, destinationFilePath)
```
[the remark this answers is not in the document]
Comment (maxwellmattryan) [reply packages/desktop/electron/electronApi.js:327]: I'll try this out, top!

File: packages/shared/features/onboardingFeatures.ts [hunk 2]
```diff
@@ -170,13 +170,13 @@ const onboardingFeatures = {
         devnet: {
             enabled: true,
             claimRewards: {
-                enabled: false,
```
Comment (nicole-obrien) [inline packages/shared/features/onboardingFeatures.ts:173]: Should not be enabled until the routes are ready for beta testing

**Review by nicole-obrien — CHANGES_REQUESTED** [review]

File: packages/shared/features/onboardingFeatures.ts (hunk 2 above)
Comment (maxwellmattryan) [reply packages/shared/features/onboardingFeatures.ts:173]: I've re-disabled the flow, so it can't be accessed currently.

**Review by nicole-obrien — APPROVED** [review]

**Pull request merged**